common/src/main/resources/index-mappings/ml_memory_container.json

```diff
@@ -0,0 +1,54 @@
+{
+  "_meta": {
+    "schema_version": 1
+  },
+  "properties": {
+    "container_id": {
+      "type": "keyword"
+    },
+    "container_name": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256
+        }
+      }
+    },
+    "description": {
+      "type": "text"
+    },
+    "created_time": {
+      "type": "date",
+      "format": "strict_date_time||epoch_millis"
+    },
+    "last_updated_time": {
+      "type": "date",
+      "format": "strict_date_time||epoch_millis"
+    },
+    "index_name": {
+      "type": "keyword"
+    },
+    "semantic_storage": {
```

Comment:
How about renaming this to be more generic ? For example, some user may want to use the vanilla index rather than K-NN.  Maybe just `storage` or other generic name?